app/src/main/java/org/jellyfin/mobile/media/MediaService.kt

```diff
@@ -160,20 +159,18 @@ class MediaService : MediaBrowserServiceCompat() {
                                 quality = 90
                             })
 
-                            val description: MediaDescriptionCompat =
-                                MediaDescriptionCompat.Builder().apply {
-                                    setMediaId(MediaItemType.Library.toString() + "|" + item.id)
-                                    setTitle(item.name)
-                                    setIconUri(Uri.parse(itemImageUrl))
-                                }.build()
+                            val description: MediaDescriptionCompat = MediaDescriptionCompat.Builder().apply {
+                                setMediaId(MediaItemType.Library.toString() + "|" + item.id)
+                                setTitle(item.name)
+                                setIconUri(Uri.parse(itemImageUrl))
+                            }.build()
 
                             MediaItem(description, MediaItem.FLAG_BROWSABLE)
                         }
 
-                    result.sendResult(views.toMutableList())
-
+                    result.sendResult(views)
                 } else {
-                    result.sendError(null)
+                    result.sendResult(emptyList())
```

Comment:
Why remove the error?

Reply:
See my comment on the PR :sweat_smile:
`sendError` isn't allowed in that function and is a wrong usage resulting in a crash. I'm actually fixing a bug here.

Reply:
oooh nice